product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Tagging Cloud Service Providers"
```

Comment:
The document is only valid for aws, not for the other cloud providers

Reply:
should we point out somewhere that this feature is *currently* only available for AWS

Reply:
What about 
```
Tagging AWS resources 
```